point.py

```diff
@@ -0,0 +1,152 @@
+import math
+from math import sqrt
+import utils
+
+class Point():
+    def __init__(self, x=0, y=0, mark=[]):
+        self.x = x
+        self.y = y
+        self.magnitude = euclidean_distance((self.x,self.y), (0,0)) 
+        self.mark = mark
+    def __add__(self, val):
+        return Point(self.x + val, self.y + val)
+
+    def __radd__(self, val):
+        return Point(self.x + val, self.y + val)
+    
+    def __mul__(self,val):
+        return Point(self.x*val, self.y*val)
+    def __rmul__(self, val):
+        return Point(self.x*val, self.y*val)
+    
+    def __neg__(self):
+        return Point(-self.x, -self.y)
+ 
+    def check_if_coincident(self,secondPoint):
```

Comment:
Thanks for the suggestion, I realized just now that it would be easy to use __eq__() to check coincidence.